contributed_definitions/NXem.nxdl.xml

```diff
@@ -171,145 +236,183 @@
+           display image and spectra with web technology visualization software.  
+           NeXus specifies only the data model, i.e. the terms and their relations.  
+           These descriptions can be implemented and stored in JSON, HDF5, XML, or HSDS,  
+           file storage, or even other formats, although HDF5 is often used.  
+         * When two- and three-dimensional geometric primitive data are stored, it is useful  
+           to write additional optional XDMF fields which support additional plotting of  
+           the data with visualization software like Paraview or Blender.  
          * Consumable results of EM characterization tasks are usually a sub-set of  
            data artifacts, as there is not an infinite amount of possible  
            electron/ion beam-specimen interactions.  
          * Images of electron counts detected in specific operation modes (bright  
-           field, dark field in TEM, secondary/back-scattered, Kikuchi in SEM)  
-         * Spectra (X-ray quanta or auger electron counts)  
-         * These data are in virtually all cases a result of some numerical
-           processing. It makes sense to name them with a controlled vocabulary,  
-           e.g. SE (secondary electron), BSE (back-scattered electron), Kikuchi,  
-           X-ray, Auger, Cathodolum(inescence) etc.  
-         
-         A key question often asked with EM experiments is how the actual (meta)data
-         should be stored (in memory or on disk). To this end the schema, here makes no
+           field, dark field in TEM, secondary/back-scattered, Kikuchi).  
+         * Spectra (X-ray quanta or Auger electron counts)  
+         * These data are in virtually all cases a result of some numerical processing. 
+           These data and processing steps are modelled as instances of :ref:`NXprocess`  
+           which use terms from a controlled vocabulary e.g. SE (secondary electron),  
+           BSE (back-scattered electron), Kikuchi, X-ray, Auger, Cathodolum(inescence).  
+         
+         **A key question often asked with EM experiments is how the actual (meta)data
+         should be stored (in memory or on disk)**. To this end the schema here makes no
          specific assumptions, not even that all the fields/group of a schema instance
-         have to be stored into a single file. Instead, the schema specifies the
-         relations between metadata, constraints on how they should be formatted, what
-         they conceptually represent and which terms (controlled vocabulary) is
-         practical to store with the data.
+         have to be stored at all into a single file. Instead, the schema specifies the
+         relations between metadata, some of the constraints and conditions on how the
+         data should be formatted, what the data conceptually represent, and which terms
+         (controlled vocabulary) is practical to store to index specific quantities.
```

Comment:
This discussion is general to file storage and facility management, not a specific problem of NXem.  In my opinion, I don't think it's needed to state it here.

Reply:
removed

Reply:
Edited in PR1271